integration_tests/tests/temporary-accommodation/manage/report.cy.ts

```diff
@@ -73,25 +75,126 @@ context('Report', () => {
       page.shouldPreselectProbationRegion(this.actingUserProbationRegion)
 
       // And when I fill out the form
+      const type = 'bookings'
       const probationRegion = this.actingUserProbationRegion
       const month = '3'
       const year = '2023'
 
       page.completeForm(month, year)
 
-      cy.task('stubReportForRegion', { data: 'some-data', probationRegionId: probationRegion.id, month, year })
+      cy.task('stubReportForRegion', {
+        data: 'some-data',
+        probationRegionId: probationRegion.id,
+        month,
+        year,
+        type,
+      })
+      page.expectDownload()
+      page.clickDownload(type)
+
+      // Then a report should have been requested from the API
+      cy.task('verifyReportForRegion', { probationRegionId: probationRegion.id, month, year, type }).then(requests => {
+        expect(requests).to.have.length(1)
+      })
+
+      // And the report should be downloded
+      const filePath = path.join(
+        Cypress.config('downloadsFolder'),
+        reportForProbationRegionFilename(probationRegion, month, year, type),
+      )
+
+      cy.readFile(filePath).then(file => {
+        expect(file).equals('some-data')
+      })
+    })
+  })
+
+  it("allows me to download an occupancy report for the acting user's region", () => {
+    // Given I am signed in
+    cy.signIn()
+
+    // When I visit the report page
+    cy.task('stubReportReferenceData')
+    const page = ReportNewPage.visit()
+
+    cy.then(function _() {
+      // Then I should see the user's probation region preselected
+      page.shouldPreselectProbationRegion(this.actingUserProbationRegion)
+
+      // And when I fill out the form
```

Comment:
Might just be me but I'm not sure about this 'and when'  
I think we should have another given, when, then. 
Maybe something like 'Given I need a report, when I fill out...'

Reply:
yeah I agree. I've tidied it up a bit